tests/view_tests/tests/test_i18n.py

```diff
@@ -314,6 +314,16 @@ def test_jsi18n_fallback_language(self):
             self.assertContains(response, 'il faut le traduire')
             self.assertNotContains(response, "Untranslated string")
 
+    def test_i18n_fallback_language_plural(self):
+        """
+        Make sure fallback to a language with less plural forms always maintains
```

Comment:
No need for "Make sure" prefix -- just state the expected behavior per [test docstring guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).